arbitrary-users-patch/Dockerfile

```diff
@@ -5,6 +5,9 @@ USER 0
 COPY --chown=0:0 entrypoint.sh /
 RUN mkdir -p /home/user && chgrp -R 0 /home && chmod -R g=u /etc/passwd /etc/group /home && chmod +x /entrypoint.sh
 
+COPY install-tooling.sh /tmp
```

Comment:
I would prefer `install-editors.sh` or `install-editor-tooling.sh`, and also a comment in the script explaining why we do it.

Reply:
Good point, done. I originally left it generic with the idea that it could be reused to ensure non-editor tooling is installed (e.g. `git`, etc.).